src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, name, location, datasetpath):
                           'primds':               {},   # Dict of primary dataset info
                           'dataset':              {},   # Dict of processed dataset info
                           'file_parent_list':     [],   # List of file parents
+                          'dataset_parent_list':  [],   # List of parent datasets
```

Comment:
I believe I mentioned it before, but why parent dataset is a list instead of a simple string?

Reply:
As we discussed I will just update the comments that dbs requires this as list.